### PR TITLE
Covid Test screen

### PR DESCRIPTION
**Pull request by JoshElgar**
# Description

This is the branch that merges Kenneth and I's Covid Test screens.
- Adds new Covid Test screens, one for listing and one for adding/updating tests.
- Added CovidTestService that extends UserService - easier to see what's going on.
- Added calendar picker component (thanks @selvach!) with our branding.
## On what platform have you tested the change?

- [ ] Android - Emulator
- [X] Android - Physical device
- [ ] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

Screenshot_20200506-094124
Screenshot_20200506-093859
Screenshot_20200506-093840

## Checklist
- [ ] I have updated mockServer

## Out of scope and potential follow-up
Bit more work on UI?

### CODE REVIEW

File: src/components/CalendarPicker.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import CalendarPicker, { CalendarPickerProps } from 'react-native-calendar-picker';
+import { colors } from '../../theme';
+import { screenWidth } from './Screen';
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+const ZoeCalendarPicker = (props: CalendarPickerProps) => (
+  <CalendarPicker
+    {...props}
+    startFromMonday={true}
```
Comment (selvach) [inline src/components/CalendarPicker.tsx:10]: American's like to start their week on Sunday.

File: assets/lang/en.json [hunk 2]
```diff
@@ -518,5 +527,15 @@
...
-  "join-total-people-contributing": "Join %{total} people contributing worldwide"
+  "join-total-people-contributing": "Join %{total} people contributing worldwide",
+
+  "add-new-test": "Add new test",
+  "your-covid-tests": "Your COVID-19 tests",
+  "positive": "Positive",
+  "negative": "Negative",
+  "failed": "Failed",
+  "pending": "Pending",
+  "please-add-test-below": "If you’ve ever had a COVID-19 test,  please add below. You can edit these at any time in the future.",
```
Comment (selvach) [inline assets/lang/en.json:538]: Is this double space intentional?

**Comment by abubuwe** [comment]
Is it worth displaying dates in the table using calendar month ("May") to prevent US/UK confusion?

File: appConfig.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // @ts-ignore
-import { API_URL } from "react-native-dotenv";
+import { API_URL } from 'react-native-dotenv';
 
+console.log('API URL: yyyzzzxxsdasdasd', API_URL);
```
Comment (abubuwe) [inline appConfig.ts:4]: should be removed.

Comment (JoshElgar) [reply appConfig.ts]: I knew I'd leave it in a commit one day

**Pull request merged**